src/pandablocks_ioc/_pvi.py

```diff
@@ -122,14 +124,15 @@ def add_automatic_pvi_info(
         if record_name == "PCAP:ARM":
             component = SignalRW(
                 name=pvi_name,
-                pv=record_name,
+                read_pv=record_name,
```

Comment:
Actually I think this has to be `write_pv`:

https://github.com/epics-containers/pvi/blob/6fb6b509c0f5297c3ae0463a79ac959529f8ac16/src/pvi/device.py#L353-L356